src/CSS/App.css

```diff
@@ -0,0 +1,21 @@
+.background {
+  position: fixed;
+  max-height: 100%;
+  min-width: 100%;
+  z-index: -1;
+  opacity: .3;
```

Comment:
You will probably want to add the light blue brackground color in your index.css file so the background will be seamless.  When I opened on a different screen, I had some whiteness showing through.